node-src/git/getChangedFilesWithReplacement.ts

```diff
@@ -39,7 +39,8 @@ export async function getChangedFilesWithReplacement(
       `Got error fetching commit for #${build.number}(${build.commit}): ${err.message}`
     );
 
-    if (/(bad object|uncommitted changes)/.test(err.message)) {
+    console.error(err.message);
```

Comment:
We'll want to pull this line before merge.

Reply:
Done!